app/src/main/java/org/eyeseetea/malariacare/domain/usecase/GetSurveyAnsweredRatioUseCase.java

```diff
@@ -79,4 +77,58 @@ public SurveyAnsweredRatio reloadSurveyAnsweredRatio(GetSurveyAnsweredRatioUseCa
         mSurveyAnsweredRatioRepository.saveSurveyAnsweredRatio(surveyAnsweredRatio);
         return surveyAnsweredRatio;
     }
+
+
+
+    public void fixTotalQuestion(QuestionDB question, boolean visible) {
```

Comment:
This use case should only has responsability of retrieve AnsweredRatio from database but should has responsability to modify it